### PR TITLE
Don't require inv params to build a dev agent that just works & updated the docs

### PR DESCRIPTION
**Pull request by albertvaka**
Makes the agent built with default inv commands work (no need to pass `--embedded-path` to inv nor to use `LD_LIBRARY_PATH` to point to dev/lib when loading the binary), just build and run with:

```
    inv six.build
    inv six.install
    inv agent.build
    ./bin/agent/agent
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ To start working on the Agent, you can build the `master` branch:
 2. cd into the project folder: `cd $GOPATH/src/github.com/DataDog/datadog-agent`.
 3. install project's dependencies: `invoke deps`.
    Make sure that `$GOPATH/bin` is in your `$PATH` otherwise this step might fail.
-4. build the whole project with `invoke agent.build --build-exclude=systemd` (with `--use-venv` to use a python virtualenv)
+4. Build the `six` dependency with `invoke six.build && inv six.install`. You will need CMake installed and a C++ compiler for this to work.
+5. build the agent with `invoke agent.build --build-exclude=systemd` (with `--use-venv` to use a python virtualenv)
```
Comment (hush-hush) [inline README.md]: `use-env` is no longer used, we could remove it from our tasks. You need to use `python-home-2` and `python-home-3`

**Review by hush-hush — CHANGES_REQUESTED** [review]

File: docs/dev/agent_dev_env.md [hunk 2]
```diff
@@ -65,7 +65,10 @@ environment and ensure a clean system python.
 ```virtualenv $GOPATH/src/github.com/DataDog/datadog-agent/venv```
 - Enable the virtual environment:
 ```source $GOPATH/src/github.com/DataDog/datadog-agent/venv/bin/activate```
-
+- Specify the path when building the agent:
```
Comment (hush-hush) [inline docs/dev/agent_dev_env.md]: you're more likely to set both to be able to switch using the conf.

File: README.md [hunk 3]
```diff
@@ -25,8 +25,10 @@ To build the Agent you need:
    This will also pull in [Invoke](http://www.pyinvoke.org) if not yet installed.
 
 **Note:** you may want to use a python virtual environment to avoid polluting your
-      system-wide python environment with the agent build/dev dependencies. By default, this environment is only used for dev dependencies listed in `requirements.txt`, if you want the agent to use the virtual environment's interpreter and libraries instead of the system python's ones,
-      add `--use-venv` to the build command.
+      system-wide python environment with the agent build/dev dependencies. You can create a virtual environment
+      using `virtualenv` and then use the `invoke` parameter  `--python-home-2=<venv_path>` or
```
Comment (hush-hush) [inline README.md]: same here it's `or`/`and`

**Review by masci — APPROVED** [review]
LGTM, thanks for fixing the docs

**Review by hush-hush — APPROVED** [review]

**Pull request merged**